pkg/discovery/testdata/podlogs/kube-system/coredns-74ff55c5b-cpfqs/logs/coredns.txt

```diff
@@ -0,0 +1,4 @@
+.:53
```

Comment:
For all these testdata files, please nest them down one extra layer to avoid polluting that directory itself. That way we have
```
testdata
|- healthTestsHappy
|- healthTestsWithErrors
|- healthTestsMissingFiles
|- otherTests
...
```